src/pages/staking/index.tsx

```diff
@@ -27,196 +25,197 @@ import StakingCommunityCallout from "../../components/Staking/StakingCommunityCa
+import { Box, Flex, Grid, Heading } from "@chakra-ui/react"
+import theme from "../../@chakra-ui/gatsby-plugin/theme"
 
-const HeroStatsWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  background: ${({ theme }) => theme.colors.layer2Gradient};
-  padding-bottom: 2rem;
-`
-
-const Page = styled.div`
-  display: flex;
-  justify-content: space-between;
-  width: 100%;
-  margin: 0 auto 4rem;
+type ChildOnlyProp = { children: ReactNode }
 
-  padding-top: 4rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-  }
-`
-
-const InfoTitle = styled.h2`
-  font-size: 3rem;
-  font-weight: 700;
-  text-align: right;
-  margin-top: 0rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    text-align: left;
-    font-size: 2.5rem
-    display: none;
-  }
-`
-
-const StyledButtonDropdown = styled(ButtonDropdown)`
-  margin-bottom: 2rem;
-  display: flex;
-  justify-content: flex-end;
-  text-align: center;
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    align-self: flex-end;
-  }
-`
-
-const InfoColumn = styled.aside`
-  display: flex;
-  flex-direction: column;
-  position: sticky;
-  top: 6.25rem; /* account for navbar */
-  height: calc(100vh - 80px);
-  flex: 0 1 330px;
-  margin: 0 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
+const HeroStatsWrapper = (props: ChildOnlyProp) => (
+  <Flex
+    direction="column"
+    alignItems="center"
+    bg="layer2Gradient"
+    pb={8}
+    {...props}
+  />
+)
 
-const MobileButton = styled.div`
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    background: ${(props) => props.theme.colors.background};
-    box-shadow: 0 -1px 0px ${(props) => props.theme.colors.border};
-    width: 100%;
-    bottom: 0;
-    position: sticky;
-    padding: 2rem;
-    z-index: 99;
-    margin-bottom: 0rem;
-  }
-`
+const Page = (props: ChildOnlyProp) => (
+  <Flex
+    justifyContent="space-between"
+    w="100%"
+    m="0 auto 4rem"
+    pt={16}
+    flexDirection={{ base: "column", xl: "row" }}
+    {...props}
+  />
+)
 
-const MobileButtonDropdown = styled(StyledButtonDropdown)`
-  margin-bottom: 0rem;
-  display: none;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: block;
-  }
-`
+const InfoTitle = (props: ChildOnlyProp) => (
+  <Heading
+    fontSize="5xl"
+    fontWeight="700"
+    textAlign="right"
+    mt="0"
+    display={{ base: "none", xl: "block" }}
+    {...props}
+  />
+)
 
-// Apply styles for classes within markdown here
-const ContentContainer = styled.article`
-  flex: 1 1 ${(props) => props.theme.breakpoints.l};
-  position: relative;
-  padding: 2rem;
-  padding-top: 0rem;
-  gap: 2rem;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    padding: 0rem;
-  }
-  .featured {
```

Comment:
This is an unused class in the component, I didn't include it in the refactored code.